app/src/main/java/chat/literacyapp/org/chat/main/MainActivity.java

```diff
@@ -0,0 +1,59 @@
+package chat.literacyapp.org.chat.main;
+
+import android.bluetooth.BluetoothDevice;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+import chat.literacyapp.org.chat.R;
+import chat.literacyapp.org.chat.main.MainFragment.Callback;
+import chat.literacyapp.org.chat.session.ChatSessionActivity;
+
+/**
+ * An example full-screen activity that shows and hides the system UI (i.e.
+ * status bar and navigation/system bar) with user interaction.
+ */
+public class MainActivity extends AppCompatActivity implements Callback {
+
+    //will use this later to make a two pane layout of list of devices, and chat for tablet
```

Comment:
@oscarmakala Instead of displaying a list of tablets, can we present a chat room shared by all devices? This way, we get a group chat instead of 1-to-1 chat sessions.